.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Fetch tags
+        run: git fetch --tags origin release
+
+      - run: make install
+
+      - name: Authenticate npm
+        run: ./bin/ci/npm-auth.sh
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Release
+        run: npx lerna publish from-git --yes
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Merge release
+        run: ./bin/ci/sync-release.sh
```

Comment:
This script is missing from this PR… 👀

Reply:
I removed this part as we said that we will be deploying by merging branch into main.